tests/hypothesis_strategies/kernels.py

```diff
@@ -0,0 +1,7 @@
+"""Hypothesis strategies for kernels."""
+
+import hypothesis.strategies as st
+
+from baybe.kernels import MaternKernel
+
+matern_strategy = st.builds(MaternKernel, st.sampled_from((0.5, 1.5, 2.5)))
```

Comment:
We adopted the hypothesis naming convention for strategies, which is "snake case plural".
```suggestion
matern_kernels = st.builds(MaternKernel, st.sampled_from((0.5, 1.5, 2.5)))
```
Of course, you need to change also downstream